packages/fxa-react/components/Portal/index.test.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect';
-
+import { render, screen } from '@testing-library/react';
 import Portal from './index';
 
-afterEach(cleanup);
+const { getByTestId } = screen;
```

Comment:
Shoot @LZoog I just remembered we had a preference discussion about destructuring screen at the top. I'm not adamant on this format, happy to adjust to `screen.getBy*` an not destructure if you'd prefer!

Reply:
Ah oops I left a comment on the first instance of this in the PR without seeing this comment lol. Yeah I mean personally, I like using `screen`, but I don't like it being destructured at the top and it being truly global😅